### PR TITLE
Fix package information

### PR DESCRIPTION
**Pull request by thechampagne**

### CODE REVIEW

File: D/DogAPI/Versions.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ["1.0.0"]
-git-tree-sha1 = "4ce09a6f533629106581683627ba990df00f9731"
+git-tree-sha1 = "7cc90eb9872302897550442f3ea0e726e901c8c8"
```
Comment (fredrikekre) [inline D/DogAPI/Versions.toml:2]: This is not allowed.

**Review by fredrikekre — CHANGES_REQUESTED** [review]

File: D/DogAPI/Versions.toml (hunk 1 above)
Comment (thechampagne) [reply D/DogAPI/Versions.toml:2]: There is no such a repo "https://github.com/thexxiv/DogAPI.jl.git"

Comment (thechampagne) [reply D/DogAPI/Versions.toml:2]: It was my old account, new repo have new git hash

Comment (thechampagne) [reply D/DogAPI/Versions.toml:2]: So If you didn't change the hash It won't be a problem to download the package ?

**Comment by thechampagne** [comment]
.

**Comment by giordano** [comment]
> .

That won't change the fact we can't accept this PR

**Comment by thechampagne** [comment]
> 
> That won't change the fact we can't accept this PR

is there a solution?

**Comment by giordano** [comment]
> is there a solution?

If the old git history isn't available anywhere, not really.

**Pull request closed without merging**